### DISCUSSION TITLE
Convert CAPs into a BIDs derivative

### DISCUSSION THREAD

**Discussion by omar-rifai** [aramis-lab/clinica#438] [Ideas]
This discussion's aim is to list everything needed to convert our CAPs data-structure into the community-driven BIDs derivative data-structure

**Comment by alexandreroutier** [comment]
Hi @omar-rifai,

You can start with this spreadsheet I created last year : https://docs.google.com/spreadsheets/d/1xv45XZY2qW5BIowjI2ztLXvfXHaaJKCePIGNsIXkXgE/edit

It listed the suffixes and keys/values for CAPS and compared them with BIDS, Derivatives and BEPs.

Best,
Alexandre
